app/models/post.rb

```diff
@@ -1,2 +1,13 @@
 class Post < ApplicationRecord
+  belongs_to :user
+  has_many :likes
+  has_many :comments
```

Comment:
- [ ] [REQUIRED] Good job so far! The `associations` between `models` should be set up correctly according to the given [ERD diagram](https://github.com/microverseinc/curriculum-rails/blob/main/blog-app/images/blog_app_erd.png). Please implement my suggestions below 🙏 

```ruby
# app/models/post.rb
belongs_to :author, class_name: 'User'
has_many :comments
has_many :likes
```